test/transactions/0_transfer.js

```diff
@@ -205,4 +206,28 @@ describe('#transfer transaction', () => {
 				.and.be.hexString();
 		});
 	});
+
+	describe('unsigned transfer transaction', () => {
+		beforeEach(() => {
+			transferTransaction = transfer({
+				recipientId,
+				amount,
+				unsigned,
+			});
+		});
+
+		it('should create a transfer transaction without signature', () => {
+			transferTransaction.should.have.property('type').equal(0);
+			transferTransaction.should.have.property('amount').equal('1000');
+			transferTransaction.should.have.property('fee').equal('10000000');
+			transferTransaction.should.have
+				.property('recipientId')
+				.equal(recipientId);
+			transferTransaction.should.have.property('senderPublicKey').equal(null);
+			transferTransaction.should.have.property('timestamp');
+			transferTransaction.should.have.property('asset').and.be.type('object');
+			transferTransaction.should.not.have.property('signature');
+			transferTransaction.should.not.have.property('id');
+		});
+	});
```

Comment:
Prefer atomic tests which reference the names rather than raw values (e.g. `amount` rather than `'1000'`).